Domains/4-Application/OpenSite/Released/OpenSite.01.00.11.ecschema.xml

```diff
@@ -0,0 +1,1490 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- ==================================================================================
+|  * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+|  * See LICENSE.md in the project root for license terms and full copyright notice.
+======================================================================================= -->
+<ECSchema schemaName="OpenSite" alias="opnsite" version="01.00.11" xmlns="http://www.bentley.com/schemas/Bentley.ECXML.3.2" displayLabel="OpenSite" description="OpenSite+ application schema.">
+    <ECSchemaReference name="CoreCustomAttributes" version="01.00.03" alias="CoreCA" />
+    <ECSchemaReference name="BisCustomAttributes" version="01.00.00" alias="bisCA"/>
+    <ECSchemaReference name="BisCore" version="01.00.10" alias="bis" />
+    <ECSchemaReference name="ClassificationSystems" version="01.00.05" alias="clsf" />
+    <ECSchemaReference name="LinearReferencing" version="02.00.03" alias="lr" />
+    <ECSchemaReference name="RoadRailUnits" version="01.00.02" alias="rru" />
+    <ECSchemaReference name="SpatialComposition" version="01.00.01" alias="spcomp" />
+    <ECSchemaReference name="StormSewerPhysical" version="01.00.00" alias="stmswrphys" />
+    <ECSchemaReference name="CivilPhysical" version="01.00.01" alias="cvphys" />
```

Comment:
Version 01.00.10 of this schema referenced CivilPhysical 01.00.02. I'm not sure if going back to a previous-version referenced is supported (cc @ColinKerr), but was this intentional?